### DISCUSSION TITLE
播放在线视频时无法截图

### DISCUSSION THREAD

**Discussion by Jadaff** [hooke007/mpv_PlayKit#361] [mpv 通用]
播放在线视频时，按 Ctrl+s（# 截屏（无字幕、无OSD） 输出原始分辨率）截图，报错：Error writing screenshot!

**Comment by hooke007** [comment]
日志

**Reply by hooke007** [reply]
你文件名长度爆了

**Reply by Jadaff** [reply]
看了一下，文件名长度都超过600个字符了，windows好像最大支持260 个字符，我用的 https://github.com/LuckyPuppy514/Play-With-MPV 油猴脚本播放的在线视频，请问有什么解决方法吗，这个是应该反馈给Play-With-MPV那边吧

**Reply by hooke007** [reply]
你把截图命名模板换个正常的不就行了

**Reply by Jadaff** [reply]
解决了，非常感谢！
